src/Microsoft.AspNetCore.Mvc.ViewFeatures/Controller.cs

```diff
@@ -383,7 +383,7 @@ public virtual void OnActionExecuted(ActionExecutedContext context)
         /// <summary>
         /// Releases all resources currently used by this <see cref="Controller"/> instance.
         /// </summary>
-        /// <param name="disposing"><c>true</c> if this method is being invoked by the <see cref="Dispose"/> method,
+        /// <param name="disposing"><c>true</c> if this method is being invoked by the <see cref="M:Dispose"/> method,
```

Comment:
Refer to `<see cref="Dispose()/>`. The method group includes the method this comment is about.